docs/lib/datastore/fragments/ios/auth-model.md

```diff
@@ -0,0 +1,97 @@
+## Updated schema
+
+The following schema is an example of a model named `Todo` that is readable and writable by the creator, and read only to everyone else.
```

Comment:
Suggestion:
Change this to:
The following schema is an example of a model named `Todo` that uses the `@auth` directive.
```
.... then show the code..
```
then explain it with something like..
By setting `allow: owner`, this schema demonstrates that owners of `Todo` objects can create, update and delete these instances.  Other users in this case will have read-only access to this object.